content/blog/tikv-3.0ga-release-notes.md

```diff
@@ -0,0 +1,55 @@
+
+* **Enhancing operator friendliness.** Human or machine, we've empowered our operators to get more out of TiKV by unifying our log format, adding new features to `tikv-ctl`, adding even more in depth metrics, and serving HTTP based metrics. This makes TiKV easier to operate, inspect, and enjoy.
+
+* **Refining request types.** While TiKV previously had commands like `BatchGet`, 3.0 brings a new `BatchCommands` request type. We've made taught TiKV how to handle batches of requests of differing kinds, leading to less data on the wire and better performance. We also added support for raw reversed scanning, and `Insert` semantics on prewrite.
+
+* **Reducing write amplication.** We were inspired by the great ideas from [WiscKey](https://www.usenix.org/system/files/conference/fast16/fast16-papers-lu.pdf) and implemented a feature we dubbed 'Titan'. It works best with larger (>1 KB) values.
+
+You can see all the changes in detail [here](https://github.com/tikv/tikv/blob/release-3.0/CHANGELOG.md).
+
+## Improved Performance
+
+Using [`go-ycsb`](https://github.com/pingcap/go-ycsb) we benchmarked TiKV 3.0.0-rc.2 against TiKV 2.1.12. We benchmarked a cluster of 3 TiKV nodes and 1 PD node. They were running on public cloud machines with 16 cores, 32 GB of RAM, a 400 GB NVMe SSD, and a 10Gbps network. We used the default settings of TiKV and PD for this.
+
+In order to achieve high concurrency and properly stress the system, we ran YCSB across three machines and set the following:
+
+* 1KB Value Size
+* 100 Field Length
+* 10 Field Count
+* 3000 Thread Count (for each machine, so a total of 9000)
+
+| Workload              | operation | 2.1.12 Avg Latency | 3.0.0-rc.2 Avg Latency | 2.1.12 QPS | 3.0.0-rc.2 QPS | QPS Diff |
+|-----------------------|-----------|--------------------|------------------------|------------|----------------|----------|
+| A 50% read/50% update | read      | 4.908              | 1.971                  | 40645.7    | 43159.2        | +6.18%   |
+|                       | update    | 206.649            | 197.140                | 40635.7    | 43186.5        | +6.28%   |
+| B 95% read/5% update  | read      | 28.121             | 13.967                 | 247164.1   | 542027.0       | +119.30% |
+|                       | update    | 132.953            | 43.307                 | 12988.6    | 28632.4        | +120.44% |
+| C 100% read           | read      | 31.056             | 14.583                 | 282752.8   | 606659.5       | +114.55% |
+| Load (10 MB)          | insert    | 198.764            | 217.901                | 44743.5    | 40819.9        | -8.77%   |
+
+Since TiKV usually is deployed with a stateless query layer, PingCAP has also published benchmarks showcases performance of TiKV paired with TiDB, the MySQL speaking query layer TiKV was originally created to compliment. You can find those benchmarks, and how to reproduce them, [**here**](https://github.com/pingcap/docs/tree/master/v3.0/benchmark).
```

Comment:
```suggestion
In the meantime, PingCAP has also published benchmarks that showcases the performance of TiKV paired with TiDB, the MySQL speaking query layer TiKV was originally created to compliment. You can find those benchmarks, and how to reproduce them, [**here**](https://github.com/pingcap/docs/tree/master/v3.0/benchmark).
```

Reply:
Suggest removing the first sentence, which looks a bit redundant in meaning and makes sentence too long.